Client/Ecosia/UI/WhatsNew/WhatsNewCell.swift

```diff
@@ -0,0 +1,70 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Core
+
+final class WhatsNewCell: UITableViewCell {
+    
+    var contentConfigurationToUpdate: Any?
+    private var imageURL: URL?
+        
+    func configure(with item: WhatsNewItem, images: Images) {
+        
+        selectionStyle = .none
+        backgroundColor = .clear
+        
+        guard let itemImageURL = item.imageURL else { return }
+        imageURL = itemImageURL
+
+        // Load the image asynchronously
+        images.load(self, url: itemImageURL) { [weak self] imageData in
+            guard let self = self else { return }
+            guard self.imageURL == imageData.url else { return }
+            let image = UIImage(data: imageData.data)
+
+            // Configure based on iOS version
+            if #available(iOS 14, *) {
+                self.configureForiOS14(image: image, item: item)
+            } else {
+                self.configureForiOS13(image: image, item: item)
+            }
+        }
+    }
+    
+    @available(iOS 14, *)
+    private func configureForiOS14(image: UIImage?, item: WhatsNewItem) {
+        var newConfiguration = defaultContentConfiguration()
+        newConfiguration.text = item.title
+        newConfiguration.textProperties.font = .preferredFont(forTextStyle: .headline)
```

Comment:
I think you additionally need to specify `adjustsFontSizeToFitWidth` to make the font adjust. Also applies for secondaryTextProperties and iOS 13 UILabel.font

Reply:
Interesting point, double checked the NewsCell and indeed does have it. Will update it too 👍